src/issue/issue.ts

```diff
@@ -45,4 +46,18 @@ export class IssueService {
       code2ProtocolConverter(currentlyOpenFileUri));
   }
 
+  analyseOpenFileIgnoringExcludes() {
+    const editor = VSCode.window.activeTextEditor;
```

Comment:
this does not check for Jupyter notebooks. We should also check `activeNotebookEditor`